src/Feature/Checkout/client/Integration/OrderHistory/sagas.ts

```diff
@@ -30,15 +30,14 @@ export function* getOrderHistory(requestData: Action<OrderHistoryRequestPayload>
   try {
     yield put(actions.GetOrderHistoryRequest());
     const firstPageNumber = 0;
-    const { data, error }: Result<Commerce.OrderHistoryResultModel> = yield call(Order.getOrders, firstPageNumber, ItemsPerPage);
+    const { data, error }: Result<Commerce.Order[]> = yield call(Order.getOrders, firstPageNumber, ItemsPerPage);
```

Comment:
```
export interface OrderHistoryResultModel {
        currentPageNumber: number;
        orders: CartModel[];
    }
```
1. Don't we actually need `currentPageNumber` anymore?
2. Now orders are also cart models?

Reply:
1. Yes, I don't know why in previous version we has it. It doesn't use at all, just passed through without changes. And actually track of which current page now is not backend task.
2. Yes, with some additional fields.